src/test/java/org/jabref/logic/importer/WebFetchersTest.java

```diff
@@ -43,7 +43,7 @@ class WebFetchersTest {
     @BeforeEach
     void setUp() {
         importFormatPreferences = mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS);
-        importerPreferences = mock(ImporterPreferences.class);
+        importerPreferences = mock(ImporterPreferences.class, Answers.RETURNS_DEEP_STUBS);
```

Comment:
Why this change? Not needed, is it?

Reply:
Thank you for your review. I have resolved your other change suggestions, but for this one, I found that if I do not add RETURNS_DEEP_STUBS, the test will fail with Null Pointer Exception from ImporterPreferences.getCatalogs(). I tried some ways, like checking if it is null and returning an empty list if it is, but it still fails with the same result. I am not familiar with Mockito but I guess it is necessary. Do you have any ideas on solving it? Thank you.